topics/connecting.md

```diff
@@ -6,6 +6,29 @@ body_class: connecting
 
 This article explains how to connect to various third party tools you might use to query Advanced MIS.
 
+### Before you can connect
+
+Before you can connect to your Advanced MIS data warehouse, you will need to supply Talis with information about which users should be allowed access.  Talis will need the following information:
+
+* User's name
+* User's email address
+* External IP Address or IP Address range (described below)
+
+#### IP Addresses
+
+We need an __externally visible__ IP Address or a range of IP Addresses for each user. We will use this information to add the user's IP Address to a firewall which restricts access to the Advanced MIS Data Warehouse to only those who should have access and reduces the chance of security issues.
+
+User workstations will need to either have a statically assigned IP Address, or be part of a known subnet range which is ideally NOT a range that covers every IP Address in the whole uni.
```

Comment:
```suggestion
User workstations will need to either have a statically assigned IP Address, or be part of a known subnet range which is ideally __not__ a range that covers every IP Address in the whole uni.
```

Bringing this into line with the other highlighted parts of this section. :)  I prefer the lower-case and bold format.